tools/osv-linter/internal/linter.go

```diff
@@ -40,6 +40,10 @@ func lint(content *Content, config *Config) (findings []checks.CheckError) {
 		log.Printf("%q: invalid JSON", content.filename)
 	}
 
+	if !checks.ValidateJSON(string(content.bytes), content.filename, config.verbose) {
+		findings = append(findings, checks.CheckError{Code: checks.CheckInvalidSchema.Code, Message: "Invalid JSON"})
```

Comment:
Should the message contain the schema that actually occured?

Though either way I think them message shouldn't be invalid JSON, since json is likely valid, just doesn't pass the schema.